spec/features/tasks/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Viewing tasks' do
```

Comment:
Where is this spec getting its data from?

Reply:
I am sorry I forgot. Added.